ulauncher/utils/Theme.py

```diff
@@ -138,6 +138,9 @@ def _get_path_for_generated_css(self):
         rmtree(new_theme_dir)
         copytree(self.path, new_theme_dir)
 
+        # change file permissions (because Nix store is read-only)
```

Comment:
probably drop the Nix store mention, and just say  it's possible a source could be read only when we expect it to be writable.

Reply:
Whoops, nice catch!